libminifi/include/io/tls/TLSSocket.h

```diff
@@ -189,9 +186,10 @@ class TLSSocket : public Socket {
   }
 
   void close_ssl(int fd);
-  std::atomic<bool> connected_;
+
+  std::atomic<bool> connected_{};
```

Comment:
I think this should be `connected_{false}`, as until C++20 no initialization takes place with the default atomic constructor: https://en.cppreference.com/w/cpp/atomic/atomic/atomic